### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**
[DEMO LINK](https://Zelinskyi-Serhii.github.io/react_keyboard/)

### CODE REVIEW

File: src/components/PushButton.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.KeyboardsListener);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.KeyboardsListener);
+  }
+
+  KeyboardsListener = (event: KeyboardEvent) => {
+    this.setState({ pushedButton: `The last pressed key is [${event.key}]` });
```
Comment (gretoq) [inline src/components/PushButton.tsx:21]: 🔥

File: src/App.tsx [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import React from 'react';
+import { PushButton } from './components/PushButton';
 
 export const App: React.FC = () => (
   <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
+    <PushButton />
```
Comment (gretoq) [inline src/App.tsx:6]: I think it's overkill for one message

**Review by gretoq — APPROVED** [review]
Gj

File: src/App.tsx (hunk 2 above)
Comment (illiarerun) [reply src/App.tsx:6]: Yep, I also think so

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**